tests/models/cohere2/test_modeling_cohere2.py

```diff
@@ -244,6 +244,7 @@ def test_export_static_cache(self):
         )
 
         model_id = "CohereForAI/c4ai-command-r7b-12-2024"
+        tokenizer = AutoTokenizer.from_pretrained(model_id)
```

Comment:
Hi @Sai-Suraj-27 Sorry for the delayed response, I missed I am pinged for this PR.
As you mentioned in your last comment, #35682 is fixed the original issue.

And currently change in this PR, after this new line

> tokenizer = AutoTokenizer.from_pretrained(model_id)

a few lines later, there is

> tokenizer = AutoTokenizer.from_pretrained(model_id, pad_token="<PAD>", padding_side="right")

So I believe we should just keep 

> tokenizer = AutoTokenizer.from_pretrained(model_id, pad_token="<PAD>", padding_side="right")

and there is no need to merge this PR, right?

Apologize again for the delay that cause this happening.